### DISCUSSION TITLE
License clarification

### DISCUSSION THREAD

**Discussion by ugintl** [Blair2004/NexoPOS#334] [General]
I have seen the license, but I am still confused. I want to use the free version for a client. What are the conditions?

**Comment by Blair2004** [comment]
Hi, you're free to use the free version. However that has some limitations :

- you cannot download modules from the marketplace
- your support is limited to GitHub issues

**Reply by ugintl** [reply]
Do I have to tell the client that it is NexoPOS? or I can sell it, modified or unmodified, as a general software to the client?

**Reply by Blair2004** [reply]
You can create a customized version that hides the brand (NexoPOS). Since you plan to redistribute the software, it should be made under the same license GPL v3.

**Reply by ugintl** [reply]
Is $40 one time lifetime payment?

**Comment by Blair2004** [comment]
Hi, yes  for further details you might consider reading the terms

https://my.nexopos.com/en/marketplace/help/terms-of-use

**Reply by ugintl** [reply]
I read it. It says that I can only use a license on single project. 

Do I have to buy every time I want to use it for a client? 

What if I buy it once and modify it and use for multiple clients? No license required

**Comment by Blair2004** [comment]
If you want all your client to benefit from our services (my.nexopos.com)
you'll need to have a license for each client. Remember NexoPOS 4.x is free
but the access to the marketplace (with extensions) is subject to the terms
I've shared above.

This open-source project however is subject to the GNU GPL V3 LICENSE. So,
it's free, you can customize it and sell it as long as you don't violate
the license (GNU GPL V3), which means the customized project should be
publicly available same as this one.

Let me know if you have any other questions.
